packages/alert-banner/index.tsx

```diff
@@ -60,7 +60,7 @@ function AlertBanner({
    * to remember the dismissal of this banner
    */
   function closeBanner() {
-    cookie.set(dismissalCookieId, 1)
+    cookie.set(dismissalCookieId, '1')
```

Comment:
🤔  was this a type error?

Reply:
Yup, I added the [types package](https://github.com/hashicorp/react-components/pull/630/files#diff-f7917ea701516a60f29f5b1677192ac32f5deaa9f624ee55e6fc752e460275b5R30) to (eventually) support UTM params from cookies, and caught this. I'll probably move this out to a separate PR though.